modules/dailymotionBidAdapter.js

```diff
@@ -0,0 +1,55 @@
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'dailymotion',
+  gvlid: 573,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   * The only mandatory parameters for a bid to be valid are the api_key and position configuration entries.
+   * Other parameters are optional.
+   *
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: () => {
+    const dmConfig = config.getConfig('dailymotion');
+    return !!(dmConfig?.api_key && dmConfig?.position);
+  },
+
+  /**
+   * Make a server request from the list of valid BidRequests (that already passed the isBidRequestValid call)
+   *
+   * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+   * @param {BidderRequest} bidderRequest
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: (validBidRequests = [], bidderRequest) => validBidRequests.map(bid => ({
+    method: 'POST',
+    url: 'https://pb.dmxleo.com',
+    data: {
+      bidder_request: bidderRequest,
```

Comment:
FYI, the structure of these request objects is not stable; major versions very often introduce breaking changes to them, and sometimes even minor updates do if the refactoring required is small enough. 

People are also not required to keep Prebid up to date so as those changes happen, you'd need not only to immediately update your server side logic but also somehow keep every previous version working.

Reply:
I realize this is probably coming too late - but if your goal is to do as much as possible server side, my recommendation is to use ortb, which is an actual protocol. The client can be [about as simple](https://github.com/prebid/Prebid.js/tree/master/libraries/ortbConverter), and while I'm not familiar with server side tooling, it's an established standard so I'd expect to find some support there as well.

Reply:
Hello @dgirardi, we can maybe continue the discussion we started on https://github.com/dailymotion-oss/Prebid.js/pull/4#discussion_r1474155769 here with the updated PR?